critiquebrainz/db/users.py

```diff
@@ -0,0 +1,51 @@
+from hashlib import md5
+import sqlalchemy
+from critiquebrainz import db
+
+
+def avatar(user):
+    """Args:
+        user: user data (dict or str)
```

Comment:
And at this point I don't see why there can't be just 3 arguments:
* id
* show_gravatar
* email

instead of the same data wrapped into a dictionary.

Again, suggested before in https://github.com/metabrainz/critiquebrainz/pull/73#discussion_r95694397.